templates/contest/list.html

```diff
@@ -148,10 +148,14 @@ <h4>{{ _('Active Contests') }}</h4>
             <table class="contest-list table striped">
                 <thead>
                 <tr>
-                    <th style="width:90%">{{ _('Contest') }}</th>
-                    <th>{{ _('Users') }}</th>
+                    <th class="contest" style="width:90%">
+                        <a href="{{ sort_links.name }}">{{ _('Contest') }}{{ sort_order.name }}</a>
+                    </th>
+                    <th class="users">
+                        <a href="{{ sort_links.user_count }}">{{ _('Users') }}{{ sort_order.user_count }}</a>
+                    </th>
                     {% if not request.in_contest %}
-                        <th style="width:15%"></th>
+                        <th class="join" style="width:15%"></th>
```

Comment:
Perhaps I'm just not finding it, but I don't think the `join` class is defined anywhere and it's not used anywhere else either where there's a `<th style="width:15%"></th>` such as [here](https://github.com/DMOJ/online-judge/blob/9858c8bf703d9950740c84bdfc3ad6d6da1c4683/templates/contest/list.html#L196) or [here](https://github.com/DMOJ/online-judge/blob/9858c8bf703d9950740c84bdfc3ad6d6da1c4683/templates/contest/list.html#L272).